database/psql.seed.js

```diff
@@ -0,0 +1,102 @@
+
+const csv = require('fast-csv');
+const fs = require('fs');
+const data = require('../database/dummy_data.js')
+
+
+const generateBooks = () =>{
+  
+  var books = {
+    title: data.title(),
+    description: data.description(),
+    author_id: data.author_id({min:1, max:1000000}),
+    published_year: data.year({ min: 1920, max: 2019}),
+    cover: data.cover + data.author_id({min:1,max: 7})+'.jpg',
+    status: data.status}
+
+    return books
+}
+
+const generateAuthor = () => {
+ 
+ var authors = {
+    name: data.name(),
+    details: data.details(),
+    profile_pic: data.profilePic(),
+    followers: data.followers({min: 0, max: 20000})
+  }
+  return authors
+}
+
+
+//Writing CSV file 
+var csvStream = csv.createWriteStream({headers: true})
+var writableStream = fs.createWriteStream("BooksData.csv");
+
+writableStream.on("finish", function(){
+  console.log("Books DONE!");
+});
+
+csvStream.pipe(writableStream);
+csvStream.on('drain', function() {
+  writeBooks();
+});
+var i =10000000;
+
+function writeBooks(){
+
+while( i>0){
```

Comment:
Maybe clean up the formatting here - looks you're missing some spaces and semi-colons in the code